docs/upgrade.md

```diff
@@ -102,6 +102,14 @@ Replace all instances of `show` and `before:show` with `render` and
 `before:render`. If you want the view to be visible in the DOM, then listen to
 the `dom:refresh` event.
 
+### Changes to `region.show()`
+
+The `region.show()` method (and by extension `showChildView()`) made a change to
```

Comment:
This isn't true.  If a view is in a region and you empty the region, it _only_ removes the el of the view.. anything else that was added extraneous to the view, should be cleaned up similarly to how it was added..  But if you show a view inside of a region that has html in it, but no view, it _will_ clear out the existing html

https://github.com/marionettejs/backbone.marionette/blob/master/src/region.js#L210